components/kyma-metrics-collector/pkg/keb/client.go

```diff
@@ -42,6 +43,8 @@ func NewClient(config *Config, logger *logrus.Logger) *Client {
 func (c Client) NewRequest() (*http.Request, error) {
 	kebURL, err := url.ParseRequestURI(c.Config.URL)
 	if err != nil {
+		totalRequest.WithLabelValues(fmt.Sprintf("%d", http.StatusBadRequest)).Inc()
```

Comment:
in this case we do not have a request at all, so also no bad request

Reply:
changed label to "failed_createRequest"